src/main/java/com/company/mvcfreemarker/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package com.company.mvcfreemarker.service;
+
+import com.company.mvcfreemarker.dto.UserDto;
+
+import java.util.List;
+
+public interface UserService {
```

Comment:
Also, maybe remove the interface? Leave just implementation class and call it UserService

Reply:
Ok. It is also created to follow dependency inversion patterns. But for this concrete sample, I think this is maybe unnecessary.